### PR TITLE
Accessibility preferences

### PR DESCRIPTION
**Pull request by piuccio**
Toggle flashing images from accessibility help page

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -146,7 +146,10 @@ module.exports = function (grunt) {
      */
     grunt.registerTask('test:unit', function(app) {
         var target = app ? ':' + app : '';
-        grunt.config.set('karma.options.singleRun', (options.singleRun === false) ? false : true);
+        if (options.singleRun === false) {
+            grunt.config.set('karma.options.singleRun', false);
+            grunt.config.set('karma.options.autoWatch', true);
+        }
```
Comment (johnduffell) [inline Gruntfile.js]: is this change related or just some tidyup?

File: applications/app/views/fragments/indexBody.scala.html [hunk 2]
```diff
@@ -3,10 +3,17 @@
 @import model.FrontProperties.empty
 @import services.IndexPage
 @import views.html.fragments.containers.facia_cards.container
+@import views.html.fragments.containers.facia_cards.containerScaffold
 @import views.html.fragments.pagination
 @import views.support.RenderClasses
 @import services.FaciaContentConvert
 
+@if(index.page.id == "help/accessibility-help") {
```
Comment (johnduffell) [inline applications/app/views/fragments/indexBody.scala.html]: seems a bit odd to have an if statement for all index pages just for this case, is it possible to have the dependency the other way to keep it out of the way?

File: Gruntfile.js (hunk 1 above)
Comment (piuccio) [inline Gruntfile.js]: It's somehow related. It was the only way to easily run unit tests while developing this feature.

File: static/src/javascripts/projects/common/modules/accessibility/helpers.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    'fastdom',
+    'common/modules/accessibility/main',
+    'common/utils/$'
+], function (
+    fastdom,
+    accessibility,
+    $
+) {
+    function hideFlashingImages(callback) {
+        if (!accessibility.isOn('flashing-images')) {
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/accessibility/helpers.js]: if isOn flashing images  then remove flashing images.  some reverse logic is confusing me there!

**Comment by johnduffell** [comment]
:+1: great work there for accessibility!

File: applications/app/views/fragments/indexBody.scala.html (hunk 2 above)
Comment (sndrs) [inline applications/app/views/fragments/indexBody.scala.html]: yeah seems like this should be a `willLoad` property or something.

more importantly, we should really have a way of rendering react-based UIs on the server...

File: static/src/javascripts/bootstraps/accessibility.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    'react',
+    'common/modules/accessibility/main'
+], function (
+    React,
+    accessibility
+) {
+    function init(callback) {
+        var BinaryToggle = React.createClass({
+            render: function () {
+                return React.DOM.div({
```
Comment (sndrs) [inline static/src/javascripts/bootstraps/accessibility.js]: totally need some JSX on this once @rich-nguyen gets cors sorted out!

**Comment by piuccio** [comment]
Renamed few things and better feedback when the option is unchecked.
